snippets/settings.py

```diff
@@ -23,6 +23,8 @@
 ALLOWED_HOSTS = config('ALLOWED_HOSTS', cast=Csv())
 ALLOWED_CIDR_NETS = config('ALLOWED_CIDR_NETS', default='', cast=Csv())
 
+ENFORCE_HOST = config('ENFORCE_HOST', default=None)
```

Comment:
I like to use `cast=Csv()` for this one (like `ALLOWED_CIDR_NETS` above) since you can have a list of enforced hosts, but it's not a requirement.